src/org/usfirst/frc2876/Steamworks2017/commands/AutoMobility.java

```diff
@@ -0,0 +1,67 @@
+package org.usfirst.frc2876.Steamworks2017.commands;
+
+import org.usfirst.frc2876.Steamworks2017.Robot;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.command.Command;
+
+public class AutoMobility extends Command {

```

Comment:
Looks like this command is driving a set distance either by PID or time.  How about making two different commands that do each thing instead of mixing the two versions in one command?
Also, the name isn't very clear as to what this class does.  Something like AutoDistance seems a little clearer.  If you end up creating two commands.. maybe AutoDisantanceTime and AutoDistance (pid is implied in class name).